### DISCUSSION TITLE
URL parameters in app routing

### DISCUSSION THREAD

**Discussion by iainhallam** [lowdefy/lowdefy#1427] [Questions and Help]
A lot of web frameworks these days give you access to routing controls that let you create URLs like `/support/tickets/ticket-reference`, which is something that's easily bookmarked. It looks to me as though Lowdefy supports only `/support/tickets?ref=reference` style, but I may be misunderstanding the documentation on this! Is the former sort of URL something that Lowdefy can support?

**Comment by SamTolmay** [comment]
Hi @iainhallam 

Currently we don't support the kind of URL parameters you are refering to.

We have discussed this and it does make sense to to add more advanced routing as a feature, but unfortunatly this is not something we will be prioritising in the near future.
